CAP/gap/InstallAdds.gi

```diff
@@ -128,6 +128,19 @@ InstallGlobalFunction( CapInternalInstallAdd,
         
         InstallMethod( ValueGlobal( install_name ), replaced_filter_list{[ 2 .. Length( replaced_filter_list ) ]}, get_convenience_function( ValueGlobal( install_name ) ) );
         
+        if IsPackageMarkedForLoading( "JuliaInterface", ">= 0.2" ) then
+            
+            if "list_of_objects" in filter_list or "list_of_morphisms" in filter_list then
+                
+                InstallOtherMethod( ValueGlobal( install_name ),
+                        CAP_INTERNAL_REPLACE_STRINGS_WITH_FILTERS_FOR_JULIA( filter_list ){[ 2 .. Length( replaced_filter_list ) ]},
```

Comment:
Please also install a version with the category as first argument (i.e. just duplicate this code and remove the sublist access).